docs/fx-core/aad-help.md

```diff
@@ -135,4 +135,24 @@ Root cause of this error is that necessary info is not included in env.default.j
 
     *Note: You also need to add secret in default.userdata file with the key in your env.default.json file as following.*
 
-    ![add secret](../images/fx-core/aad/skip-secret.png)
\ No newline at end of file
+    ![add secret](../images/fx-core/aad/skip-secret.png)
+
+
+## aad.AadGetAppError
+
+### Error Message
+
+Failed to get app info with current Object Id in env.default.json. Please make sure object id is valid, or delete 'objectId' under fx-resource-aad-app-for-teams in env.default.json and try again.
+
+### Mitigation
+
+Root cause of this error is that toolkit can not find Azure AD app with the same object id saved in your "env.default.json" file. Please follow the instruction following to address the error.
+
+1. Open `.fx\env.default.json` file
+2. Find `fx-resource-aad-app-for-teams`. Note value of key *clientId* or *local_clientId*
+3. Go to Azure Portal, login with the same account as the M365 account in toolkit, select "Azure Active Directory"
+4. Select "App Registrations" and search for you Azure AD app by client id noted above.
+
+If you can find your Azure AD app, please check your network status and try again.
+
+If you can not find your Azure AD app, please check whether you logged in with the correct account.
```

Comment:
Do we need to add another tip that: "You can also remove objectId from env.default.json file"

Reply:
Fixed. Thanks!